src/scripts/main.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(`${eventName}`, () => {
+      // eslint-disable-next-line max-len
```

Comment:
why you need it?

Reply:
try to remove lint error about length of row